packages/relay/src/lib/clients/sdkClient.ts

```diff
@@ -587,12 +587,7 @@ export class SDKClient {
       const transactionRecord: TransactionRecord = await transactionResponse.getRecord(this.clientMain);
 
       // get transactionFee and gasUsed for metrics
-      /**
-       * @todo: Determine how to separate the fee charged exclusively by the operator because
-       *        the transactionFee below includes the entire charges of the transaction,
-       *        with some portions paid by tx.from, not the operator.
-       */
-      transactionFee = transactionRecord.transactionFee.toTinybars().toNumber();
+      transactionFee = getTransferAmountSumForAccount(transactionRecord, this.clientMain.operatorAccountId?.toString());
```

Comment:
Is it ever possible for `operatorAccountId` to be `null` or `undefined` here? 

If it's not possible, we can directly do:
```suggestion
      transactionFee = getTransferAmountSumForAccount(transactionRecord, this.clientMain.operatorAccountId!.toString());
```